pkg/controller/plan/adapter/ova/builder.go

```diff
@@ -31,6 +31,10 @@ const (
 	BIOS = "bios"
 )
 
+const (
+	UEFI = "uefi"
+)
```

Comment:
this should be placed next to BIOS above - you can replace `BIOS types` with `Firmware types`